tests/test-cases/Group1-Docker-Commands/1-06-Docker-Run.robot

```diff
@@ -125,6 +125,9 @@ Docker run verify container start and stop time
     Run Keyword Unless  ${startStatus}  Fail  container start time before command start
     ${stopStatus}=  Run Keyword And Return Status  Should Be True  ${cmdStart} < ${containerStop}
     Run Keyword Unless  ${stopStatus}  Fail  container stop time before command start
+    ${timeDiff}=  Evaluate  ${containerStop}-${cmdStart}
+    ${timeLimit}=  Run Keyword And Return Status  Should Be True  0 < ${timeDiff} < 60000
+    Run Keyword Unless  ${timeLimit}  Fail  start to stop time limit reached, over 60 seconds
```

Comment:
This looks good, just a bit redundant I think, which adds complexity in reading it. You can really just rely on the Should be True keyword and get rid of the entire 3rd line and half the second line.